webvtt.go

```diff
@@ -692,3 +703,30 @@ func cssColor(rgb string) string {
 	}
 	return colors[strings.ToLower(rgb)] // returning the empty string is ok
 }
+
+// webVTTColorMap maps WebVTT color class names to Color instances
+var webVTTColorMap = map[string]*Color{
```

Comment:
Can you rename the function to `func newColorFromWebVTTString(s string) (c *Color, err error)` and use a `switch` instead?